### PR TITLE
Remove serverless kyma ci

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- remove jobs related to serverless from kyma repository

**Related issue(s)**
https://github.com/kyma-project/test-infra/issues/8991

### CODE REVIEW

File: templates/data/generic_component_data.yaml [hunk 1]
```diff
@@ -75,367 +75,3 @@ templates:
...
-                    - "--config=/config/kaniko-build-config.yaml"
-                    - "--context=tests/serverless-bench"
-                    - "--dockerfile=Dockerfile"
-                inheritedConfigs:
-                  global:
-                    - kaniko_buildpack
-                    - jobConfig_postsubmit
-                    - "disable_testgrid"
-
-      - to: ../../prow/jobs/kyma/tools/gitserver/gitserver-generic.yaml
```
[the remark this answers is not in the document]
Comment (dbadura) [reply templates/data/generic_component_data.yaml:404]: Removed the generated file

**Review by anoipm — APPROVED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**